person.rb

```diff
@@ -0,0 +1,23 @@
+require './person'
+
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
```

Comment:
- hey, to improve the readability of your code and to follow the best convention in ruby, kindly move this line up before the constructor.